site/package.json

```diff
@@ -16,6 +16,8 @@
     "lint:types": "tsc -p .",
     "playwright:install": "playwright install --with-deps chromium",
     "playwright:test": "playwright test --config=e2e/playwright.config.ts",
+    "//": "If we are in a coder workspace, use the `ui-port` to allow opening remotely. If native, use the default app. It's generally a bit smoother to use.",
+    "playwright:test-ui": "playwright test --config=e2e/playwright.config.ts --ui $([[ \"$CODER\" == \"true\" ]] && echo --ui-port=7500 --ui-host=0.0.0.0)",
```

Comment:
@aslilac This works on local or in coder.

Reply:
why not just set these flags all the time? they're harmless options and seem like they should always work.